app/requirements.txt

```diff
@@ -26,4 +26,14 @@ persistqueue
 retry
 PyPDF2
 pytz
-aiosqlite
\ No newline at end of file
+aiosqlite
+starlette
+torch
+langchain~=0.0.141
+nltk
+numpy
+requests
+python-dateutil
+httplib2
+pypdf
+pycryptodome
```

Comment:
are you sure all of these needed?

Reply:
@Roey7 yes they are required by pypdf and langchain